File: app/src/main/java/com/elementary/tasks/backups/UserItem.java
package com.elementary.tasks.backups;

/**
 * Copyright 2016 Nazar Suhovich
 * <p/>
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 * <p/>
 * http://www.apache.org/licenses/LICENSE-2.0
 * <p/>
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

public class UserItem {

    private String name;
    private String photo;
    private long quota;
    private long used;
    private long count;
    private UserInfoAsync.Info kind;

    public UserItem() {
    }

    public UserItem(String name, long quota, long used, long count, String photo) {
        this.name = name;
        this.quota = quota;
        this.used = used;
        this.count = count;
        this.photo = photo;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public long getQuota() {
        return quota;
    }

    public void setQuota(long quota) {
        this.quota = quota;
    }

    public long getUsed() {
        return used;
    }

    public void setUsed(long used) {
        this.used = used;
    }

    public long getCount() {
        return count;
    }

    public void setCount(long count) {
        this.count = count;
    }

    public UserInfoAsync.Info getKind() {
        return kind;
    }

    public void setKind(UserInfoAsync.Info kind) {
        this.kind = kind;
    }
}

File: app/src/main/java/com/elementary/tasks/backups/UserItem.kt
package com.elementary.tasks.backups

/**
 * Copyright 2016 Nazar Suhovich
 *
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

class UserItem {

    var name: String = ""
    var photo: String = ""
    var quota: Long = 0
    var used: Long = 0
    var count: Int = 0
    var kind: UserInfoAsync.Info? = null

    constructor()

    constructor(name: String, quota: Long, used: Long, count: Int, photo: String) {
        this.name = name
        this.quota = quota
        this.used = used
        this.count = count
        this.photo = photo
    }
}
